app/controllers/tasks_controller.rb

```diff
@@ -9,6 +9,11 @@ def create
         end
     end
 
+    def index
+        tasks = Task.all 
+        render json: tasks
```

Comment:
Nit - instead of defining a variable that is only used once, you could just do something like `render json: Task.all`

Reply:
Make sense, done!